src/components/BulkEnrollmentPage/stepper/AddLearnersStep.jsx

```diff
@@ -11,7 +11,6 @@ import { logError } from '@edx/frontend-platform/logging';
 import { Link } from 'react-router-dom';
 import { BulkEnrollContext } from '../BulkEnrollmentContext';
 import { ADD_LEARNERS_TITLE } from './constants';
-import { convertToSelectedRowsObject } from '../helpers';
```

Comment:
Go ahead and delete `convertToSelectedRowsObject` entirely, if you don't mind. These were the only to components using it.